go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/emicklei/dot
```

Comment:
this change should not have been committed

Reply:
sorry, was by mistake. I will undo this commit.